www-src/package.json

```diff
@@ -30,7 +30,9 @@
     "pouchdb": "5.1.0",
     "stylus-brunch": "^1.8.1",
     "uglify-js-brunch": "> 1.0 < 1.5",
-    "underscore": "1.4.4"
+    "underscore": "1.4.4",
+    "backbone.statemachine": "0.2.7",
```

Comment:
Ok, I didn't know (I used shortcut -s which doesn't exist). But be careful  on the other Cozy applications, we use only strict version number ("1.2.3", no "^1.2.3" or ranges or ...), it was decided to improve app quality, and I think optimize installation speed.

About cozy-mobile dependencies, could we move all dev related dependencies to devDependencies, keeping the app dependency more readable ie :
```
    "async": "1.2.1",
    "backbone": "1.0.0",
    "backbone.statemachine": "0.2.7",
    "jquery": "1.9.1",
    "moment": "2.10.6",
    "moment-timezone": "0.4.1",
    "node-polyglot": "1.0.0 ",
    "pouchdb": "5.1.0",
    "semver": "5.1.0",
    "underscore": "1.4.4"